libs/packages/components/src/lib/autocomplete-search/autocomplete-search.component.ts

```diff
@@ -243,7 +243,7 @@ export class SDSAutocompleteSearchComponent implements ControlValueAccessor {
     }
   }
   isCleanIconVisible(): boolean {
```

Comment:
Is this function name a typo? Should this be `isClearIconVisible()`?

Reply:
Updated